app/jobs/cleanup_votes_job.rb

```diff
@@ -0,0 +1,17 @@
+class CleanupVotesJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    Community.all.each do |c|
+      RequestContext.community = c
+      orphan_votes = Vote.all.reject { |v| v.post.present? }
+      puts "[#{c.name}] destroying #{orphan_votes.length} #{'orphan vote'.pluralize(orphan_votes.length)}"
```

Comment:
Just as a safety net, in case we ever accidentally destroy a valid vote (maybe I'm being paranoid): can we record the rows we're nuking somewhere (output, log file, whatever)?  This seems very unlikely, but also, a user saying "wait where'd all my votes go???" would be bad.

Reply:
Sure, I can add a log to the successful case of `destroy` as well, won't hurt (even though we definitely should have backups to restore from even if something extra gets dropped).

Reply:
We do have backups, yes.  I was thinking vaguely that it might be helpful to be able to assess the scale of the problem (like how many users were affected).